docs/source/tutorial.md

```diff
@@ -16,6 +16,11 @@ $ tox -e py3 -- test/integration/e2e_test.py # Run specified test case
 $ deactive
 ```
 
+If denied permission to connect to Docker daemon socket, run:
+```bash
+$ sudo chmod 666 /var/run/docker.sock
```

Comment:
I think this fix should be in the FAQ rather than the docs as `sudo` is not required by the sdk